package.json

```diff
@@ -2385,6 +2385,207 @@
                                             }
                                         ]
                                     },
+                                    {
+                                        "key": "languageId:c",
+                                        "name": "C",
+                                        "type": "bindings",
+                                        "bindings": [
+                                            {
+                                                "key": "h",
+                                                "name": "Describe thing at point",
+                                                "icon": "book",
+                                                "type": "command",
+                                                "command": "editor.action.showHover"
+                                            },
```

Comment:
Seems like this is not specific to c nor it is in the spacemacs?

Reply:
I was referring #301 and saw this feature, looked to be useful. But I don't mind taking this off.

Reply:
I see. Since it is not really in spacmeacs' C bindings any way. Maybe consider taking it off? Also. `g h` from vim also does this.

Reply:
Ooh, sure. I've made the change